src/store/storeProfile/storeProfileThunks.ts

```diff
@@ -1,9 +1,17 @@
 import $api from '@/http';
 import { createAsyncThunk } from '@reduxjs/toolkit';
-import { setStore, setName, setDescription, setShopPhoto, setBannerPhoto, setStorePhoneNumber, setLink } from './storeProfileSlice';
+import {
+  setStore,
+  setName,
+  setDescription,
+  setShopPhoto,
+  setBannerPhoto,
+  setStorePhoneNumber,
+  setLink,
+} from './storeProfileSlice';
 
 export const getStoreInfo = createAsyncThunk(
-  'shops/info',
+  'storeSettings/info',
```

Comment:
It's a good idea to change the name, but it should be done in a separate branch.

Reply:
will do next time =)